docs/alternatives/navigator_extension_alternative.bs

```diff
@@ -58,10 +58,30 @@ Markup Shorthands: markdown yes, css no
 
 ### {{Navigator/getEnvironmentIntegrity()}} ### {#navigator-getenvironmentintegrity}
 
-<div class="algorithm" data-algorithm="navigator-getenvironmentintegrity-alg">
+<div algorithm="navigator-getenvironmentintegrity-alg">
+    The browser has a global <dfn for="attestations sessions map">attestation sessions map</dfn>,
+    which is a [=map=] that contains each attestation session. An attestation session contains any
```

Comment:
You need to define in a Concept/Infrastructure section what an attestation session is, and its lifetime.

You can also define the methods to interact with it for the purpose of the spec.

Here's an examples: https://www.w3.org/TR/push-api/#push-subscription

If you'd rather iterate on this, that's alright for now but please add a TODO

Reply:
Thanks this made a lot of sense.

I added temporary WebIDLs for what we need and will continue to build on it so I added a TODO.